acceptance-tests/src/acceptance-test/java/tech/pegasys/teku/test/acceptance/ValidatorLivenessAcceptanceTest.java

```diff
@@ -59,6 +60,7 @@ public void setup() {
    *  - by epoch 5, all validators should be seen as performing duties in epoch 3
    */
   @Test
+  @Disabled
```

Comment:
nit: Probably worth adding a reason here to save us going searching for the PR in the future.